packages/bierzo-wallet/src/routes/payment/components/CurrencyToSend/index.tsx

```diff
@@ -91,13 +93,13 @@ export const CurrencyToSend = ({ positionedClass }: Props) => {
       <Avatar classes={avatarClasses}>
         <FontAwesomeIcon icon={faUser} className={classes.avatarIcon} />
       </Avatar>
-      <label className={classes.sendLabel}>You send</label>
-      <div className={classes.currencyContainer}>
+      <InputLabel className={classes.sendLabel}>You send</InputLabel>
```

Comment:
This `InputLabel` can be replaced by `Typography` from `medulas-react-components`. If for any reason you need a label as final dom element, you can use Typography's component prop [see here](https://next.material-ui.com/api/typography/) and specify label there.